feature_vector.py

```diff
@@ -0,0 +1,75 @@
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+import pandas as pa
+from sklearn.feature_extraction.text import CountVectorizer
+import csv
+
+def to_lower(sentence):
+    filtered_sentence = []
+    for text in sentence:
+        temp = []
+        for w in word_tokenize(text):
+            temp.append(w.lower())
+        filtered_sentence.append(' '.join(temp))
+    return filtered_sentence
+
+def remove_stopwords(sentence):
+    stop_words = set(stopwords.words('english'))
+    filtered_sentence = []
+    for text in sentence:
+        temp = []
+        for w in word_tokenize(text):
+            if w not in stop_words:
+                temp.append(w)
+        filtered_sentence.append(' '.join(temp))
+    return filtered_sentence
+
+def nonletter_removal(sentence):
+    filtered_sentence = []
+    for text in sentence:
+        temp = []
+        for w in word_tokenize(text):
+            if w.isalpha():
+                temp.append(w)
+        filtered_sentence.append(' '.join(temp))
+    return filtered_sentence
+
+def getNgrams(filtered_sentence,n):
+    counts = dict()
+    for i in range(len(filtered_sentence) - n):
+        grams = [' '.join(filtered_sentence[i:i + n])]
+    for gram in grams:
+        if gram not in counts:
+            counts[gram] = 1
+        else:
+            counts[grams]+=1
+    with open('bigramcount.csv','w+') as f:
+        w = csv.writer(f)
+        w.writerows(counts.items())
+    f.close()
+    return
+
+def NumericalFeatureVector(processed_text):
```

Comment:
This one is a red flag for me!

```suggestion
def numericalFeatureVector(processed_text):
```
By your definition it looks like you were working with a class, and that could be confusing at first sight. Something as simple as using the correct convention can solve the entire issue.

Of course, it feels a bit weird that you have a method called `numericalFeatureVector`. What does it do? Maybe it should be called `computeNumericalFeatureVector` instead, but it's up to you.